src/lmrtfy.js

```diff
@@ -44,5 +43,27 @@ class LMRTFY {
 	}
 }
 
-Hooks.on('ready', LMRTFY.ready)
-Hooks.on('getSceneControlButtons', LMRTFY.getSceneControlButtons)
\ No newline at end of file
+Hooks.on('ready', () => {
+    LMRTFY.ready;
```

Comment:
This line is just referencing the `ready` function, but not calling it.. which is probably why everything broke.
Also, I would not modify the `ready` Hook. I'd rather just move the code below into the existing `LMRTFY.ready` function.

Reply:
Yeah, I just noticed that now - I am no longer passing the function on. Is "init" a good place for these? I remember the reason I stuffed them here is that when I put them in the ready function they ended up undefined.

Reply:
They were possibly undefined if you had used `this.something` instead of `LMRTFY.something`, as `this` is not set to the class within that ready function. Proper way to have it set would be to do `LMRTFY.ready.bind(LMRTFY)` or just `() => LMRTFY.ready()` as the hook function.
Should work now though 👍